Gesture_control.py

```diff
@@ -8,6 +8,34 @@
 import cv2
 import numpy as np
 
+
+def get_trackbars():
+    cv2.namedWindow('Trackbars')
+    cv2.resizeWindow('Trackbars', 640, 240)
+    cv2.createTrackbar('HueMin', 'Trackbars', 0, 179, empty)
+    cv2.createTrackbar('HueMax', 'Trackbars', 0, 179, empty)
+    cv2.createTrackbar('SatMin', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('SatMax', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('ValMin', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('ValMax', 'Trackbars', 0, 255, empty)
+
+def create_mask():
+    while True:
```

Comment:
You can remove this while loop because there is a while loop to read every video frame already in the code. 
You just have to call this create_mask function passing that frame in the while loop only